ui/raidboss/data/03-hw/raid/a11s.txt

```diff
@@ -26,7 +26,7 @@ hideall "--sync--"
 40.6 "Spin Crusher" sync /:Cruise Chaser:1A85:/
 
 # skippable
-45.8 "E.D.D. Add" sync / 03:........:Added new combatant E\.D\.D\./ window 50,5
+45.8 "E.D.D. Add" sync / 03:........:Added new combatant E.D.D./ window 50,5
```

Comment:
Can you leave the `E\.D\.D\.` and change your replacement sync to `E\\\\.D\\\\.D\\\\.`? (sorry)

Reply:
It's ok.